### DISCUSSION TITLE
How to stop --show

### DISCUSSION THREAD

**Discussion by ybonzou** [open-mmlab/mmsegmentation#2805] [General]
Hello, How to stop --show
The demo with "--show" launches the viewer, how can I stop the inference?
Please excuse the elementary question.
thank you for all

**Comment by xiexinch** [comment]
Hi @ybonzou,
It is a normal python process, and you can kill it like any program.

**Comment by ybonzou** [comment]
Hi, @xiexinch !! 
thank you for your reply. 
I had to close the virtual environment. 
Is it possible to exit without closing the virtual environment?
